lint-configs/eslint-rules/truth-helpers-imports.mjs

```diff
@@ -0,0 +1,29 @@
+export default {
+  meta: {
+    type: "suggestion",
+    docs: {
+      description: "use the shorter form of truth-helpers import",
+    },
+    fixable: "code",
+    schema: [], // no options
+  },
+
+  create(context) {
+    return {
+      ImportDeclaration(node) {
+        if (/truth-helpers\/helpers\//i.test(node.source.value)) {
+          context.report({
+            node,
+            message: `It is recommended to use 'truth-helpers' import instead of '${node.source.value}'.`,
+            fix(fixer) {
+              return fixer.replaceText(
+                node,
+                `import { ${node.specifiers[0].local.name} } from 'truth-helpers';`
```

Comment:
What if someone had an old import like

```
import hbsOr from "truth-helpers/helpers/or";
```
Ideally the autofix would do

```
import { or as hbsOr } from "truth-helpers";
```
or we could just skip autofixing this case, but keep the lint failure.

(this is particularly important for things like the gjs codemod, because we've seen cases where the truth-helper is imported as `or0`)